frontend/src/container/TracesExplorer/ListView/utils.tsx

```diff
@@ -36,7 +52,11 @@ export const getListColumns = (
 					typeof item === 'string'
 						? dayjs(item).format('YYYY-MM-DD HH:mm:ss.SSS')
 						: dayjs(item / 1e6).format('YYYY-MM-DD HH:mm:ss.SSS');
-				return <Typography.Text>{date}</Typography.Text>;
+				return (
+					<BlockLink to={getTraceLink(item)}>
```

Comment:
the styles will be changed here and is it expected ?

Reply:
Yes, IMO, we need the change in style i.e. `display:block` to make the whole cell clickable rather than only the text.